stdpopsim/slim_engine.py

```diff
@@ -648,6 +697,68 @@ def slim_array_string(iterable, indent, width=80):
     )
 
 
+def _simulate_mutation_type_in_slim(mt):
```

Comment:
I don't think this function helps?

Reply:
maybe not? it felt better to me to have this explicitly somewher and then if in the future we decide to change this behavior then we will have to change it just once?